src/json-uri.cpp

```diff
@@ -35,7 +35,7 @@ void json_uri::update(const std::string &uri)
 			}
 
 			std::string hex = pointer.substr(pos + 1, 2);
-			char ascii = (char) std::strtoul(hex.c_str(), nullptr, 16);
+			char ascii = static_cast<char>( std::strtoul(hex.c_str(), nullptr, 16));
```

Comment:
The spaces after the `(` is too much. Fix all of them using clang-format if you can.